app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -1,17 +1,111 @@
 "use client";
 import PageLayout from "../../pageLayout/pageLayout";
-import React from "react";
-import SearchResults from "../../search/results";
+import React, { useState, useRef, useEffect } from "react";
 import {
   Box,
   Heading,
   HorizontalRule,
   SearchBar,
+  Menu,
+  Text,
+  Pagination,
 } from "@nypl/design-system-react-components";
+import { useSearchParams, usePathname, useRouter } from "next/navigation";
 import { headerBreakpoints } from "../../../utils/breakpoints";
+import { CardsGrid } from "../../grids/cardsGrid";
+import LaneLoading from "../../lane/laneLoading";
+import {
+  displayResults,
+  totalNumPages,
+  createQueryStringFromObject,
+} from "../../../utils/utils";
+import type { SyntheticEvent } from "react";
 import { createAdobeAnalyticsPageName } from "@/src/utils/utils";
+import NoResultsFound from "../../results/noResultsFound";
 
 export const CollectionsPage = ({ data }) => {
+  const { push } = useRouter();
+  const searchParams = useSearchParams();
+  const pathname = usePathname();
+  const [isLoaded, setIsLoaded] = useState(false);
+  let collections = [];
+
+  if (data.headers.message === "Collections retrieved successfully") {
```

Comment:
This can be brittle if the text changes; the frontend doesn't own it and if the backend changes this string, it will cause errors. Is it not possible to check if `data.collections` is null, undefined, or empty?

When there's a Search model/class, there should be a flag for this such as `data.hasCollections`. For now, make this a variable so you're not testing the string in the if statement.

Make sure to write a test for `getCollectionsData` to reflect what the API returns and this message there.

Reply:
I agree, and I also think this should be moved to the page level anyways (where the 404 is checked).